src/garden/home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Routes from './Routes'
+import { AgreementSubscriptionProvider } from '../providers/AgreementSubscription'
+import { AppStateProvider } from '../providers/AppState'
+
+import { StakingProvider } from '../providers/Staking'
+import { ConnectProvider as Connect } from '../providers/Connect'
+import { useDAO } from '../providers/Dao'
+
+import { DAONotFound } from '../errors'
+
+function GardenHome({ match }) {
+  const { connectedDAO } = useDAO()
+
+  if (!connectedDAO) {
+    throw new DAONotFound(match.params.daoId)
+  }
```

Comment:
Thinking that probably this should be handled in the dao provider as we will have many routes with the dao address

Reply:
ah no, i needed to remove it from the dao provider to remove the `if(path=home)` we need to let the thing go through until the specific garden tree, because the dao provider needs to be on the general gardens tree view because of the wallet component

Reply:
Ah yeah i get what you are saying but we could bypass it checking if we got a match in the `useRouteMatch` for example 

```
export function DAOProvider({ children }) {
  const match = useRouteMatch('/garden/:daoId')

  // get all the daos for the dashboard

  // note that i am calling this daoId because at some point we might want to also handle aragon id's and not just addresses

  let connectedDao = {
    address: '',
  }

  if (match) {
    const daoAddress = match.params.daoId
    connectedDao = daoList.daos.find(d => addressesEqual(daoAddress, d.address))

    if (!connectedDao) {
      throw new DAONotFound(daoAddress)
    }
  }

  return (
    <DAOContext.Provider value={connectedDao}>{children}</DAOContext.Provider>
  )
}
```